Shared/Samples/Identify features in WMS layer/IdentifyFeaturesInWMSLayerView.swift

```diff
@@ -0,0 +1,123 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct IdentifyFeaturesInWMSLayerView: View {
+    /// A map with a dark gray base basemap centered on the USA.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISDarkGrayBase)
+        let center = Point(x: -95.7129, y: 37.0902, spatialReference: .wgs84)
+        map.initialViewpoint = Viewpoint(center: center, scale: 7e7)
+        return map
+    }()
+    
+    /// The WMS layer with EPA water info.
+    @State private var waterInfoLayer: WMSLayer?
+    
+    /// The text of a WMS feature HTML attribute that is shown in the web view.
+    @State private var webViewText = ""
+
+    /// The tapped screen point.
+    @State private var tapScreenPoint: CGPoint?
+    
+    /// The placement of the callout on the map.
+    @State private var calloutPlacement: CalloutPlacement?
+    
+    /// The string text for the identify layer overlay.
+    @State private var overlayText = "Tap on the map to identify features in the WMS layer."
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            MapView(map: map)
+                .callout(placement: $calloutPlacement.animation(.default.speed(2))) { _ in
+                    ScrollView(.horizontal) {
```

Comment:
I see a bug on iPhone where the scroll view doesn't scroll on the first swipe. Do you know if this was present when the web view was in the status bar?

https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/ac473da0-ddac-47fb-9ad3-e9268684647e

Reply:
Looks like it scrolls as it should after disabling the WebView.